services/TransmittalService.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Transmittal = require('../models/transmittal');
+
+module.exports = {
+    isValidTimestamp: function(activityYear, respondentId, timestamp, callback) {
```

Comment:
It's standardized in the file spec...

> Format is century, year, month, day, hour,
minute (e.g., Jan. 17, 2014, at 1:30 pm would be
201401171330)